localization/ndt_scan_matcher/src/map_update_module.cpp

```diff
@@ -28,29 +28,48 @@ MapUpdateModule::MapUpdateModule(
 
   pcd_loader_client_ =
     node->create_client<autoware_map_msgs::srv::GetDifferentialPointCloudMap>("pcd_loader_service");
+
+  secondary_ndt_ptr_.reset(new NormalDistributionsTransform);
+
+  if (ndt_ptr_) {
```

Comment:
Is it correct to think that the case where `ndt_ptr_` is false is a rare case? At that time, I felt that it would be good to output an ERROR log.

Reply:
@SakodaShintaro I added a `RCLCPP_ERROR_STREAM_THROTTLE` to report an error if ndt_ptr is null in the constructor of `MapUpdateModule`.